guides/project-service-management-guide.md

```diff
@@ -251,6 +251,136 @@ application repositories
+  -b, --helm-config-branch <helm-branch>                      bedrock custom helm chart configuration branch. --helm-chart-* and --helm-config-* are exclusive; you may only use one. (default: "")
+  -p, --helm-config-path <helm-path>                          bedrock custom helm chart configuration path. --helm-chart-* and --helm-config-* are exclusive; you may only use one. (default: "")
+```
+
+As noted by the the documentation text, `helm-chart-*` and `helm-config-*` are
+both mutually exclusive configurations: you can _only_ use one set of
+configurations or the other.
+
+This section intends on documenting the various use cases for both sets of
+mutually exclusive configurations.
+
+##### Helm Charts in a well-known Helm Repository
+
+A Helm Repository is a well known set of helm charts conforming to the
+[helm repository guidelines](https://helm.sh/docs/topics/chart_repository/).
+Perhaps the best known helm repository is the community run
+[helm charts repository](https://github.com/helm/charts).
+
+As an `spk` user, if you would like to incorporate helm charts from a well known
+public repository, you may simply run `spk` the following `helm-chart`
+arguments:
+
+```
+spk service create --helm-chart-chart stable/nginx --helm-chart-repository github.com/helm/charts
+```
+
+##### Helm Charts in a distinct Git Repository from Application Sources in the same Azure DevOps Project
+
+If your Helm Charts are in their own distinct Git Repository in the _same_ Azure
+DevOps project, you can use the `helm-config` arguments to configure `spk`:
+
+```
+spk service create --helm-config-git https://dev.azure.com/fabrikam/fabrikam-project/_git/fabrikam-helm-charts --helm-config-branch master --helm-path /charts/fabrikam
+```
+
+The above invocation presumes that the helm chart repository configured for
+`spk` is _different_ from the application repository configured for `spk` usage.
+
+The `helm-config-git` parameter _must not_ contain the username portion of a
+url. If you retrieve the URL from Azure DevOps's "Clone Repository" UI, it will
+automatically have the username filled for an HTTPs clone ie:
+
+`https://fabrikam@dev.azure.com/fabrikam/fabrikam-project/_git/fabrikam-helm-charts`
+
+Ensure that the you _remove_ the `fabrikam@` portion of the URL when passing
+parameters to `spk service create --helm-config-git`:
+
+`https://dev.azure.com/fabrikam/fabrikam-project/_git/fabrikam-helm-charts`
+
+##### Helm Charts in a distinct Git Repository from Application Sources in a different Azure DevOps Project
+
+If your Helm Charts are in their own distinct Git Repository in a different
+Azure DevOps project, you can use still use the `helm-config` arguments to
+configure `spk`, but must also provide another option,
+`--helm-config-access-token-variable`. This configuration option is the name of
```

Comment:
Might want to reference how in the happy-path workflow, the default value that gets injected of `ACCESS_TOKEN_SECRET` will work be default for all projects in their org (assuming they've given that permission to their token)